pkgs/applications/misc/kitty/default.nix

```diff
@@ -67,6 +67,7 @@ buildPythonApplication rec {
     (substituteAll {
       src = ./fix-paths.patch;
       libstartup_notification = "${libstartup_notification}/lib/libstartup-notification-1.so";
+      libcanberra = "${libcanberra}/lib/libcanberra.so";
```

Comment:
An alternative to this patch would be to set LD_LIBRARY_PATH in the kitty wrapper below. Note that `.so` is not valid on macOS.

Reply:
Also, libcanberra is not needed on macos: https://github.com/kovidgoyal/kitty/blob/8184ba246a63b05cd79e742c48f5527ac29c8a67/docs/build.rst#dependencies

Reply:
> Also, libcanberra is not needed on macos

Yeah, I know but neither is `libstartup_notification`, which is only used on X11 and it's handled the same way. I'm a total Nix noob, so I just went with what was already there. I don't know how to set `LD_LIBRARY_PATH` to the proper value in the wrapper. Any concrete help on how to improve this patch would be appreciated. Also feel free to submit a PR yourself if you like.

Reply:
I can take a closer look at this today. Would you provide a simple way how to reproduce this for that purpose? :)

Reply:
Awesome.
Start kitty and execute `tput bel` inside kitty. This will cause kitty to try to play a sound for which it will try to load the library.

Reply:
Updated the PR :)